scripts/pi-hole/js/network.js

```diff
@@ -113,12 +116,15 @@ $(document).ready(function() {
              // Set number of queries to localized string (add thousand separators)
              $("td:eq(6)", row).html(data["numQueries"].toLocaleString());
 
-            // Client -> jump to Query Log on click
-            $("td:eq(0)", row).click( function () { openInNewTab("/admin/queries.php?client="+this.innerHTML); } );
-            $("td:eq(0)", row).css("cursor","pointer");
-            $("td:eq(0)", row).hover(
-              function () { this.title="Click to show recent queries made by "+this.innerHTML; this.style.color="#72afd2"; },
-              function () { this.style.color=""; } );
```

Comment:
Was this functionality purposefully removed?

Reply:
Yes. We can add it back in if you think it is necessary, however, it'd need to be checked how to implement it now that we can have multiple IPs in the same line.